### ISSUE TITLE
Update play services to 11.4.2

### ISSUE THREAD

**Issue by samtipton** [RoverPlatform/rover-android#167]
When I compile against rover with other play service libraries (including location) at 11.4.2 I encounter a duplicate class error in gms/internal. Is it safe to compile this way:

```
compile ('io.rover.library:rover:1.12.4') {
    exclude group:"com.google.android.gms"
}
```
?

Either way, you should be targeting most up-to-date release to provide maximum support I think.

**Comment by samtipton** [comment]
When I compile against rover with other play service libraries (including location) at 11.4.2 I encounter a duplicate class error in gms/internal. Is it safe to compile this way:        
```
  compile ('io.rover.library:rover:1.12.4') {
            exclude group:"com.google.android.gms"
        }
```?

Either way, you should be targeting most up-to-date release to provide maximum support I think.

**Comment by seanrucker** [comment]
Yes, that is the recommended way to avoid that issue. Some of our other customers are doing it exactly as you propose.

Gradle dependencies are broken, we have yet to find a way to make this work turnkey for all use cases. Even requiring the latest version "+" has its own problems as it will always use the latest version installed on your system which may or may not be the version your app requires.

Long story short, yes, use the exclude group as you have posted will solve this issue for you.

**Comment by samtipton** [comment]
Thanks!
